src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js

```diff
@@ -7,10 +7,13 @@ import {
 } from '../resource-filter-builder/resourceFilterBuilder';
 
 const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
```

Comment:
This function now builds filters specifically for 1 day before and 2 days after the specified date.
So we probs need to change this function name to represent that it's giving us entries on **& near** the date.
Or make the name represent a range and pass in the date boundaries.